### DISCUSSION TITLE
Custom Build host OS

### DISCUSSION THREAD

**Discussion by Janrupf** [community/community#25860] [Actions]
<p>Currently there doesn’t seem to be a way to use another OS than the ones given by Github. However, I would like to use Arch Linux to build my software since Ubuntu fails horribly to provide a clean build environment. I can understand that it might not be possible to provide every OS the user wants, but maybe it could be possible to run an entire job in a docker container?</p>

**Comment by chrispat** [comment, marked as answer]
<p>You can specify a container to run your job in using the container property <a href="https://help.github.com/en/articles/workflow-syntax-for-github-actions#jobsjob_idcontainer" rel="nofollow noopener">https://help.github.com/en/articles/workflow-syntax-for-github-actions#jobsjob_idcontainer</a>.</p>
